docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,323 @@
+    type of block.
+
+  * display_name: (string) The display name of the block.
+
+  * children: (list) If the block has child blocks, a list of IDs of the child
+    blocks. Returned only if ``children`` is included in the ``requested_fields``
+    query parameter.
+
+  * block_counts: (dict) For each block type specified in the ``block_counts``
+    parameter to the endpoint, the aggregate number of blocks of that type for
+    this block and all of its descendants. Returned only if the ``block_counts``
+    query parameter contains this block's type.
+
+  * graded: (boolean) Whether or not the block or any of its descendants is
+    graded. Returned only if ``graded`` is included in the ``requested_fields``
+    query parameter.
+
+  * format: (string) The assignment type of the block. Possible values can be
+    ``Homework``, ``Lab``, ``Midterm Exam``, and ``Final Exam``. Returned only if
+    ``format`` is included in the ``requested_fields`` query parameter.
+
+  * student_view_data: (dict) The JSON data for this block. Returned only if the
+    ``student_view_data`` query parameter contains this block's type.
+
+  * student_view_url: (string) The URL to retrieve the HTML rendering of this
+    block's student view. The HTML can include CSS and Javascript code. This
+    field can be used in combination with the ``student_view_multi_device``
+    field to determine whether a block can be viewed on a device. This URL can
+    be used as a fallback if the ``student_view_data`` for this block type is
+    not supported by the client or by the block.
+
+  * student_view_multi_device: (boolean) Whether or not the block whose URL is
+    indicated in ``block_url`` can be viewed on mobile devices. Returned only
+    if ``student_view_multi_device`` is included in the ``requested_fields``
+    query parameter.
+
+  * lms_web_url: (string) The URL to the navigational container of the XBlock
+    on the web LMS. This URL can be used as a fallback if the
+    ``student_view_url`` and the ``student_view_data`` fields are not
+    supported.
+
+  * lti_url: (string) The block URL for an LTI consumer.
+
+
+============================================================================
+Example Response Showing a List of Course Blocks in a Specified Course
+============================================================================
+
+.. code-block:: json
+
+ {
+   "root": "i4x://edX/DemoX/course/Demo_Course",
+   "blocks": {
+      "i4x://edX/DemoX/course/Demo_Course": {
+        "display_name": "edX Demonstration Course",
+        "graded": false,
+        "student_view_url": "https://courses.edx.org/xblock/i4x://edX/DemoX/
+         course/Demo_Course",
+        "student_view_multi_device": false,
+        "lms_web_url": "https://courses.edx.org/courses/edX/DemoX/Demo_Course/
+          jump_to/i4x://edX/DemoX/course/Demo_Course",
+        "type": "course",
+        "id": "i4x://edX/DemoX/course/Demo_Course"
+        }
```

Comment:
To give a better idea, can we provide an example that includes other optional fields, such as `block_counts` and `student_view_data`?